### ISSUE TITLE
Updated screens on Zeplin

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#106]
Here, we will keep track of any changes to Zeplin in a centralised way.

**Comment by asimonw** [comment]
* 'Differentiate admin from users in the contact list'  https://zpl.io/Z6mADN
* Admin nav open state  https://zpl.io/2bTRLg
* Landing screen https://zpl.io/2lo2qR

**Comment by asimonw** [comment]
@JMurphyWeb @mk4111 see first few changes above, first one based on a question you asked yesterday, second one to clarify admin navigation and third design tweaks to 'home screen'.

**Comment by fannyBkane** [comment]
• Latest version of 'Browse', including latest version of bottom navigation for 'non admin users', which now includes icons https://zpl.io/1NP3uu

**Comment by mk4111** [comment]
@asimonw and @fannyBkane , thank you for the updated screens! 
In the wireframs for differentiate admin from users, could you also let us know whether you would like to be able to visually differentiate between active and inactive users too, and if so, how please?

Thank you!

**Comment by asimonw** [comment]
Good point. @fannyBkane we were thinking just green and red dots on the left, right?

**Comment by fannyBkane** [comment]
That's right @asimonw Just updated Zeplin:https://zpl.io/Z1zOzzw

**Comment by asimonw** [comment]
Here's the user flow, work in progress. Admin flow only. Primary user flow coming up.
https://collaborate.uxpin.com/45d32478d5af5fa220c1569332ec4908bb787865#/pages/57374200/sitemap

**Comment by fannyBkane** [comment]
Here is the primary user flow: https://collaborate.uxpin.com/d55e95d23abb78d24ee150b9853a0c54e2e62894

Some screens missing, will complete by early next week.

**Comment by JMurphyWeb** [comment]
This is great @fannyBkane and will be extremely useful for us. Thanks!

**Comment by asimonw** [comment]
Note that there might still be a few paths either missing or incorrect (especially when a screen can be arrived at in different ways). We will go through both flows more carefully on Monday.

**Comment by fannyBkane** [comment]
• Browse (included a tree structure for the tags instead as a simple list) 'default' state https://zpl.io/1cNDiV and 'open' state https://zpl.io/1sVV8l
• Browse-select-tag (included same change as above–a tree structure for the tags instead as a simple list) 'default' state https://zpl.io/Z1skr1k and 'open' state https://zpl.io/Z1k3PK4
• Browse-select-list (highlight colour changed to match Browse-select-tag) https://zpl.io/Z4YNjs

**Comment by fannyBkane** [comment]
• Public view of profile (Changed  layout of challenge cards, and size of headers): https://zpl.io/19CkQG
• Private view of profile (Changed  layout of challenge cards, and size of headers): https://zpl.io/Z231EN8
• Suggested match-follow up actions: (added tags): https://zpl.io/Z1Dqki5
• Challenge view (added Org): https://zpl.io/Z2kmBO6
• Browse-challenge view (removed description & added org): https://zpl.io/1yNKUh
• Error message: https://zpl.io/ZTjXWs

• Private view of profile-desktop: https://zpl.io/25v9kV
• Suggested match-follow up actions-Desktop: https://zpl.io/FP485
• Public view of profile-desktop: https://zpl.io/6ruQG
• Public view of profile-tablet:https://zpl.io/8G26X

**Comment by JMurphyWeb** [comment]
Thanks for updates and documenting here @fannyBkane 👍

**Comment by fannyBkane** [comment]
• Login screen updated with new copy... https://zpl.io/ZnMkwr
